tests/flytekit/unit/extras/tasks/testdata/test.csv

```diff
@@ -0,0 +1,4 @@
+SN,Name,Contribution

```

Comment:
should we remove this file?

Reply:
we need this file now.
currently this test.csv file is used in [test_shell.py](https://github.com/jasonlai1218/flytekit/blob/add-validation-flyteformat-type/tests/flytekit/unit/extras/tasks/test_shell.py#L17-L19)
https://github.com/jasonlai1218/flytekit/blob/add-validation-flyteformat-type/tests/flytekit/unit/extras/tasks/test_shell.py#L17-L19

The reason why I adjusted this file is that although its file extension is .csv, it is not actually a csv file format, so I adjusted it to the csv format type.

Reply:
i see, thanks